### PR TITLE
CVE-2015-1241

### PR DESCRIPTION
**Pull request by TLezynski**

### CODE REVIEW

File: cves/CVE-2015-1241.yml [hunk 1]
```diff
@@ -32,7 +32,8 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+    On mobile, click events were able to fire outside of the scope that they were created in. This can be exploited
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: How can this be exploited? Any examples?

File: cves/CVE-2015-1241.yml [hunk 2]
```diff
@@ -56,14 +57,14 @@ fixes_vcc_instructions: |
...
-  :note: ''
+  :note: 'Gesture reset logic native to Android is causing the issue. Detection isnt reset for timeout events. Will use a specific gesture detection reset hook to allieviate this.'
 - :commit: 9d343ad2ea6ec395c377a4efa266057155bfa9c1
-  :note: ''
+  :note: 'Cancel all actice touch events whenever a window is out of focus or a page is navigated away from, before the touch release event.'
 vccs:
-- :commit:
-  :note: 
+- :commit: 3c85191dc27cc4734458ff94324ffacf40f44c22
+  :note: Commit was made and introduced the vulnerability, without knowing that it was a vulnerability.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: Note should be the commit message

File: cves/CVE-2015-1241.yml [hunk 3]
```diff
@@ -84,7 +85,7 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
+  answer: Yes, there were unit tests. Both were unit tested, but additional unit testing wasn't necessary. All they did was expect "false" where they were previously expecting "true". It was introduced because it wasn't forseen as a vulnerability but it ended up that it was.
   code:
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: Set code to "true" and fix to "false" (if they didn't update or add any tests)

File: cves/CVE-2015-1241.yml [hunk 4]
```diff
@@ -102,10 +103,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer: 
-  date: 
-  automated: 
-  google: 
+  answer: It seems that the bug was discovered accidentally. The discoverer explains a reproduction case that isn't exactly perfectly reproducable, for instance, the phrase if the conditions are just right.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: What did the discoverer do to find the bug?

File: cves/CVE-2015-1241.yml [hunk 5]
```diff
@@ -125,7 +126,7 @@ interesting_commits:
     emerging themes?
 
     If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: There really weren't any interesting commits. Which is a shame.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: Anything that happened between the VCC and the fix?

File: cves/CVE-2015-1241.yml [hunk 6]
```diff
@@ -139,12 +140,12 @@ major_events:
...
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: The commit that introduced this vulnerability directly contradicts this vulnerability.
   events:
-  - name: 
-    date: 
-  - name: 
-    date: 
+  - name: Introduced
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: Don't need to put introduced/fixed here (the VHP website captures that data elsewhere).  See if there are any major discussions or milestones that happened on certain dates when trying to fix the bug or before it was even discovered.

File: cves/CVE-2015-1241.yml [hunk 7]
```diff
@@ -205,4 +206,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: This seems most like a requirements mistake. The commit that introduced the vulnerability seemed to have introduced it carelessly without knowing the consequences of what they changed. Proper requirements could have cleared up the misunderstanding. It could have been a miscommunication. I'm still wondering why it was introduced. It seems like it was just changed for seemingly no reason, just to be changed back once it caused an issue. I'd like to know what was going through the original developers heads. Maybe there is something that I missed or that I'm misunderstanding.
```
Comment (CatherineOsadciw) [inline cves/CVE-2015-1241.yml]: Remove first-person references.  What kind of requirements might have been added to  prevent this?

File: cves/CVE-2015-1241.yml (hunk 1 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: Commit incoming with the fix for this

File: cves/CVE-2015-1241.yml (hunk 2 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: We discussed this, and determined this isn't necessarily the case.

File: cves/CVE-2015-1241.yml (hunk 3 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: Easy fix, thanks for catching that.

File: cves/CVE-2015-1241.yml (hunk 4 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: It's not really noted anywhere, but I'll be a little bit more verbose in my description.

File: cves/CVE-2015-1241.yml (hunk 5 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: To be honest, as far as the commits, not very much, but I'll be more verbose in the file.
Hope you're enjoying these emails.

File: cves/CVE-2015-1241.yml (hunk 6 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: Found some interesting implementation discussions

File: cves/CVE-2015-1241.yml (hunk 7 above)
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: Thanks for the advice!

File: cves/CVE-2015-1241.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: Thanks for the feedback. I still came up short though. Seems like there was a discussion that wasn't necessarily a commit about the proposed implementation of that system. I added some more information in the file about that, but that's unfortunately all that can be done here.

File: cves/CVE-2015-1241.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TLezynski) [reply cves/CVE-2015-1241.yml]: I've addressed it now, thanks for the feedback

**Pull request merged**